### ISSUE TITLE
Draft of booklet for comment

### ISSUE THREAD

**Issue by DavidMc1948** [thecapitalistcycle/setup#2]
[Link](https://docs.google.com/document/d/1oxG-TlPs8cCwLagCEegjyEiECtrHoSQtt-SPCtMu3xc/edit?usp=sharing)

 Parts of it are lifted from my earlier booklet but the story line is a bit different. It is 10,000 words whereas the old one was 8,000. My intention is to print a fairly large number of paper copies and send it to lots of people around the world and locally.  This would be mainly people who are renowned for one reason or another. This includes "public intellectuals", political figures, journalists and so on.  There would be an associated web site which would include among things an electronic version of the booklet and a discussion forum.

**Comment by DavidMc1948** [comment]
New Conclusion

Conclusion

At present, the proletariat slumbers and a revolutionary movement does not exist even in embryo. However, as we have seen, Marxists still have things to do in the meantime. They will be loud in their support of science and innovation, and opposition to all forms of backwardness. They believe history is too slow not too fast.

They are also tasked with joining the battle to defend the Marxist worldview.  Here they will first have to cross swords with the "Marxians" (or is that Martians?) who have managed to embed themselves in academe. 

Marxists see our communist future as the beginning of the real human journey. We will enter a world where we can achieve the precondition for the thriving of each - the thriving of others.

**Comment by dentarthur** [comment]
will read soon. still collecting tech stuff.

google docs link seemed to be same as printout. not changed initial para
you read on phone.

**Comment by DavidMc1948** [comment]
I've stopped revising the google doc. Now making changes to a Word document. Apart from the first few paragraphs and the conclusion, there are no major changes.

First few paragraphs:

Those who interpret Karl Marx have the primary job of ensuring that people benefit as little as possible from what he had to say. In part, this means “forgetting” a lot of it.  Here we focus on an important case, namely, Marx’s view of the historical role of capitalism.  This can be summed up in the following two points:

- Capitalism creates the necessary conditions for a more advanced classless society where we will all jointly own the means of production and the typical individual will thrive for the first time.  
- However, it is then up to us to take advantage of this opportunity and make it happen by transforming ourselves and society. 

In other words, history is providing us with a chance to make something of ourselves but we have to rise to the occasion.

**Comment by DavidMc1948** [comment]
New version with new first paragraph and conclusion. Other small edits

https://docs.google.com/document/d/1HlXv6V9ciw2oQABDMqjTFdW-Mqiaz0e7KxOZJ1cjtf0/edit?usp=sharing

**Comment by dentarthur** [comment]
Copy is "view only" so not downloaded but have printed to pdf.

Still trying to sort out LwDITA first.

**Comment by DavidMc1948** [comment]
Various small edits at Google docs

**Comment by DavidMc1948** [comment]
Arthur. I have set it so you can comment.

https://docs.google.com/document/d/1HlXv6V9ciw2oQABDMqjTFdW-Mqiaz0e7KxOZJ1cjtf0/edit?usp=sharing
